dbcon/mysql/ha_calpont.cpp

```diff
@@ -33,6 +33,8 @@ static int calpont_commit(handlerton* hton, THD* thd, bool all);
 static int calpont_rollback(handlerton* hton, THD* thd, bool all);
 static int calpont_close_connection ( handlerton* hton, THD* thd );
 handlerton* mcs_hton;
+char cs_version[256];
+char cs_commit_hash[256];
```

Comment:
IIRC a git hash is 40 characters, so is this overkill?

Reply:
I wonder if we can get away including columnstoreversion.h and using its variables as the source of the status vars?

Reply:
I arbitrarily chose the length to 256 earlier. Updated this part to set the length of cs_commit_hash to 41 and cs_version to 25 (hoping 25 is enough length for the version).

The only available status variable types are C-style strings (SHOW_CHAR). Using the variables of columnstoreversion.h directly is therefore not possible as they are defined as C++ string objects.